src/backend/backend-agnostic.md

```diff
@@ -2,11 +2,12 @@
 
 <!-- toc -->
 
-As of <!-- date-check --> Aug 2022, `rustc_codegen_ssa` provides an
-abstract interface for all backends to implement, to allow other codegen
-backends (e.g. [Cranelift]).
+[`rustc_codegen_ssa`]
+provides an abstract interface for all backends to implement,
+to allow other codegen backends (e.g. [Cranelift]).
 
 [Cranelift]: https://github.com/bytecodealliance/wasmtime/tree/HEAD/cranelift
```

Comment:
What do you think about changing this to link to Rust's cranelift backend rather than the wasmtime repo? I think in its existing state it might be a bit confusing.

Reply:
I thought that would be a better link target as well. I do it.

Reply:
see if this is resolved with https://github.com/rust-lang/rustc-dev-guide/pull/1606/commits/dfb468a52fb606b6647e7240756f2c399654f179